S2VX.Game/Editor/UserInterface/MetadataPanel.cs

```diff
@@ -0,0 +1,103 @@
+﻿using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.UserInterface;
+using osuTK;
+using osuTK.Graphics;
+using S2VX.Game.Story.Settings;
+
+namespace S2VX.Game.Editor.UserInterface {
+    public class MetadataPanel : OverlayContainer {
+        private static Vector2 InputSize = new(200, 30);
+        private static Vector2 PanelSize { get; } = new(330, 230);
+        private static Vector2 PanelPosition = new(0, S2VXGameBase.GameWidth / 2);
+        private const float Pad = 10;
+
+        private string StoryDirectory { get; }
+
+        public BasicTextBox TxtTitle { get; private set; }
+        public BasicTextBox TxtArtist { get; private set; }
+        public BasicTextBox TxtAuthor { get; private set; }
+        public BasicTextBox TxtDescription { get; private set; }
+        public BasicButton BtnSave { get; private set; }
+
+        public MetadataPanel(string storyDirectory) => StoryDirectory = storyDirectory;
+
+        private FillFlowContainer Form { get; set; } = new() {
+            Direction = FillDirection.Vertical,
+            Position = new(Pad),
+            Size = PanelSize,
+            Child = new SpriteText { Text = "Metadata Panel" }
+        };
+
+        [BackgroundDependencyLoader]
+        private void Load() {
+            Origin = Anchor.CentreLeft;
+            Position = PanelPosition;
+            Size = PanelSize;
+
+            var metadata = MetadataSettings.Load(StoryDirectory);
+            TxtTitle = AddRow("TitleTitle", metadata.SongTitle);
```

Comment:
Double title lol

Reply:
Fixed, probably committed on accident when I was showing the real time code changes